tox.ini

```diff
@@ -37,14 +37,14 @@ skip_install = True
 [testenv:pycodestyle]
```

Comment:
(a similar suggestion here) we should merge these two environments into one called `quality`